weltyt.py

```diff
@@ -4,3 +4,22 @@
 print("Welcome to tyton")
 conn = sqlite3.connect("TYTDB.db")
 
+cur = conn.cursor()
+
+def initializationDB(cur2):
```

Comment:
Move method to the beggining of the file.
No need to call the argument `cur2`. You can just name it `cur` or `cursor`.